internal/kratos/kratos.go

```diff
@@ -39,6 +41,40 @@ func New(d dependencies) Client {
 	return &Default{dependencies: d}
 }
 
+func (k *Default) Authenticate(ctx context.Context, name, secret string) (err error) {
```

Comment:
One test? :) We already have some integrations with kratos right?

Reply:
Will add!

Reply:
done